entries/s/sendcloud.com.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Example Site": {
+    "domain": "sendcloud.com",
+    "url": "https://account.sendcloud.com",
+    "img": "sendcloud.com.png",
```

Comment:
SVG files are proffered over PNG files. Here's one that you can use instead:
![svg-icon](https://user-images.githubusercontent.com/3535780/160884009-cd316f36-5e58-4ef7-b8d3-eacae9f9b3b3.svg)
```suggestion
```